### PR TITLE
Fish 5722 mes in payara resources xml

### PR DESCRIPTION
**Pull request by aubi**
## Description
Allow definition of concurrency resources to be defined in payara-resources.xml

### New tests
Added test in appserver/tests/payara-samples/samples/resources/mes-in-payara-resources-example

### Testing Environment
OpenJDK, Linux

## Documentation
TBD

### CODE REVIEW

**Comment by aubi** [comment]
Jenkins test please

**Comment by aubi** [comment]
Jenkins test please

**Comment by aubi** [comment]
Jenkins test please

File: appserver/connectors/descriptors/src/main/resources/glassfish/lib/dtds/payara-resources_1_6.dtd [hunk 1]
```diff
@@ -93,7 +93,9 @@
     ((custom-resource | external-jndi-resource | jdbc-resource | mail-resource 
     | admin-object-resource | connector-resource
     | resource-adapter-config | jdbc-connection-pool      
-    | connector-connection-pool | work-security-map)*)>                                              
+    | connector-connection-pool | work-security-map
```
Comment (Pandrex247) [inline appserver/connectors/descriptors/src/main/resources/glassfish/lib/dtds/payara-resources_1_6.dtd:96]: Hmm, I wonder if we should be making a 1.7 DTD for this.

File: appserver/resources/resources-connector/src/main/java/org/glassfish/resources/admin/cli/ResourcesXMLParser.java [hunk 2]
```diff
@@ -699,6 +731,246 @@ private void generateJDBCResource(Node nextKid, String scope) throws Exception {
...
+        Node hungaftersecondsNode = attributes.getNamedItem(HUNG_AFTER_SECONDS);
+        if (hungaftersecondsNode != null) {
+            String hungaftersecondsValue = hungaftersecondsNode.getNodeValue();
+            managedExecutorServiceResource.setAttribute(HUNG_AFTER_SECONDS, hungaftersecondsValue);
+        }
+        Node corepoolsizeNode = attributes.getNamedItem(CORE_POOL_SIZE);
+        if (corepoolsizeNode != null) {
+            String corepoolsizeValue = corepoolsizeNode.getNodeValue();
+            managedExecutorServiceResource.setAttribute(CORE_POOL_SIZE, corepoolsizeValue);
+        }
```
Comment (Pandrex247) [inline appserver/resources/resources-connector/src/main/java/org/glassfish/resources/admin/cli/ResourcesXMLParser.java:792]: Duplicate of 749-752?

File: appserver/resources/resources-connector/src/main/java/org/glassfish/resources/admin/cli/ResourcesXMLParser.java [hunk 3]
```diff
@@ -699,6 +731,246 @@ private void generateJDBCResource(Node nextKid, String scope) throws Exception {
...
+        Node hungaftersecondsNode = attributes.getNamedItem(HUNG_AFTER_SECONDS);
+        if (hungaftersecondsNode != null) {
+            String hungaftersecondsValue = hungaftersecondsNode.getNodeValue();
+            managedScheduledExecutorServiceResource.setAttribute(HUNG_AFTER_SECONDS, hungaftersecondsValue);
+        }
+        Node corepoolsizeNode = attributes.getNamedItem(CORE_POOL_SIZE);
+        if (corepoolsizeNode != null) {
+            String corepoolsizeValue = corepoolsizeNode.getNodeValue();
+            managedScheduledExecutorServiceResource.setAttribute(CORE_POOL_SIZE, corepoolsizeValue);
+        }
```
Comment (Pandrex247) [inline appserver/resources/resources-connector/src/main/java/org/glassfish/resources/admin/cli/ResourcesXMLParser.java:882]: Duplicate of 843-847?

**Review by Pandrex247 — CHANGES_REQUESTED** [review]

**Comment by aubi** [comment]
Jenkins test please

File: appserver/resources/resources-connector/src/main/java/org/glassfish/resources/admin/cli/ResourcesXMLParser.java (hunk 3 above)
Comment (Pandrex247) [reply appserver/resources/resources-connector/src/main/java/org/glassfish/resources/admin/cli/ResourcesXMLParser.java:879]: Not resolved?
```suggestion
```

**Review by Pandrex247 — CHANGES_REQUESTED** [review]
Missed a duplicate block

**Comment by aubi** [comment]
Jenkins test please

**Review by Pandrex247 — APPROVED** [review]

**Pull request merged**